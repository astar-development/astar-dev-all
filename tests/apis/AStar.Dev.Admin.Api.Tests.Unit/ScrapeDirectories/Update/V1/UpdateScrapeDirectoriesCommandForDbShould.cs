using AStar.Dev.Api.Usage.Sdk.Metrics;
using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Admin.Api.ScrapeDirectories.Update.V1;

// [TestSubject(typeof(UpdateScrapeDirectoriesCommandForDb))]
public sealed class UpdateScrapeDirectoriesCommandForDbShould
{
    [Fact]
    public void ContainTheExpectedValues()
    {
        var sut = new UpdateScrapeDirectoriesCommandForDb(Guid.Empty, new(), new());

        sut.ToJson().ShouldMatchApproved();
    }

    [Fact]
    public void ImplementIEndpointNameInterface()
    {
        var sut = new UpdateScrapeDirectoriesCommandForDb(Guid.Empty, new(), new());

        sut.ShouldBeAssignableTo<IEndpointName>();
    }
}
