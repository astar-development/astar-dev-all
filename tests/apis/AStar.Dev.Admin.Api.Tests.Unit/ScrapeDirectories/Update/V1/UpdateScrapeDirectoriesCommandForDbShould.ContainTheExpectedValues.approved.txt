{
  "scrapeDirectoryId": "00000000-0000-0000-0000-000000000000",
  "rootDirectory": "",
  "baseSaveDirectory": "",
  "baseDirectory": "",
  "baseDirectoryFamous": "",
  "subDirectoryName": "",
  "updatedBy": "me"
}